crates/std_detect/Cargo.toml

```diff
@@ -25,6 +25,9 @@ maintenance = { status = "experimental" }
 libc = { version = "0.2", optional = true, default-features = false }
 cfg-if = "0.1.10"
 
+[target.'cfg(target_os="windows")'.dependencies]
+winapi = {version = "0.3.8", features = ["processthreadsapi"]}
```

Comment:
can this depend on aarch64 too?

Reply:
winapi uses on windows/aarch64 only. Should I add additional condition?

Reply:
cc @alexcrichton @retep998: this would make `winapi` a dependency of `libstd`, but IIUC this is currently not the case:

* license of winapi looks compatible with the standard runtime
* winapi has a build.rs that we'd need to make sure works within rust's build system
* winapi might not be worth pulling into libstd for this particular feature only due to other issues (build times, dependency, etc.)

IIRC, pulling a new dependency into libstd requires the libs team to sign on on it. If you both think this is worth doing, I can try to find answers to these questions, and we can have a mini FCP on it.

Reply:
Currently I believe `winapi` being a dependency of `std` is blocked on upgrading `winapi`'s minimum supported Rust version which will happen in `winapi` 0.4.

Reply:
Thanks @retep998, I think we can land this as is, and refactor it into using `winapi` once it becomes a dependency of `std`. I'll open a tracking issue in this repo to not forget about this once this is merged.